CODEOWNERS

```diff
@@ -0,0 +1,25 @@
+# BitGoJS code owners and first officers will automatically be requested for review whenever a pull request touches the files they own.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners and first officers will be the default owners for everything in the repo.
+*   tyler@bitgo.com
+
+# Order is important. The last matching pattern has the most precedence.
+src/v2/recovery.js danny@bitgo.com
+src/v2/coins/xlm.js luis@bitgo.com
+src/v2/coins/txlm.js luis@bitgo.com
+src/v2/coins/bsv.js paddy@bitgo.com
+src/v2/coins/tbsv.js paddy@bitgo.com
+src/v2/coins/bch.js paddy@bitgo.com
+src/v2/coins/tbch.js paddy@bitgo.com
+src/v2/coins/zec.js jorge@bitgo.com
+src/v2/coins/tzec.js jorge@bitgo.com
+src/v2/coins/dash.js dawid@bitgo.com
+src/v2/coins/tdash.js dawid@bitgo.com
+src/v2/coins/xrp.js alex@bitgo.com
+src/v2/coins/txrp.js alex@bitgo.com
```

Comment:
Please set xrp and txrp as luis@bitgo.com.
We should use owners and first officers from https://bitgoinc.atlassian.net/wiki/spaces/ENG/pages/537264239/Coins+Team+Summary